user_test.go

```diff
@@ -0,0 +1,206 @@
+package goscaleio
+
+import (
+	"errors"
+	"math"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	types "github.com/dell/goscaleio/types/v1"
+	//"github.com/stretchr/testify/assert"
```

Comment:
please remove the commented code, if not needed.

Reply:
Done